target/linux/ath79/dts/qca9558_airtight_c-65.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/*
+ * This board has two identical flash ICs but only one is defined in this file.
+ * This board has one more LED for 5Ghz WiFi but it is onboard with QCA9882 (ath10k) and not defined in this file.
+ */
+
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	model = "AirTight C-65";
+	compatible = "airtight,c-65", "qca,qca9558";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &power_amber;
+		led-failsafe = &power_amber;
+		led-upgrade = &power_amber;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power_amber: power_amber {
+			label = "c-65:amber:power";
+			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		lan {
+			label = "c-65:green:lan";
+			gpios = <&gpio 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		wlan2g {
+			label = "c-65:green:wlan2g";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;

```

Comment:
Please try this with `compatible = "gpio-keys";` which is interrupt based variant, and remove that `poll-interval` property

Reply:
Done.